diktat-rules/src/test/resources/test/paragraph3/empty_block/EmptyBlockExpected.kt

```diff
@@ -16,3 +16,6 @@ fun goo () {
     while (x > 0)
         --x
 }
+
```

Comment:
I guess this time we need to have more tests

Reply:
Well, cases with NPE have been discovered during check of our own code with diktat and NPEs were present in snapshot CI build. Now they are gone at least for `diktat-common`. @kentr0w have you run check locally for other modules? Please do and comment if there are any errors left in this rule.

Reply:
I tested on other modules and there were no similar errors.